crates/client/rpc/src/lib.rs

```diff
@@ -245,6 +247,9 @@ where
             })?
             .map_err(|e| {
                 error!("Failed to call function: {:#?}", e);
+                if e == PalletError::<Runtime>::ContractNotFound.into() {
+                    return StarknetRpcApiError::ContractNotFound;
+                }
```

Comment:
I would rather have implemented it using the `From` trait.
Is there a reason you cannot use it there?

Reply:
This way you can use the `?` keyword